### DISCUSSION TITLE
Проект еще поддерживается?

### DISCUSSION THREAD

**Discussion by aalexmora** [VisualGadget/Mercutel#1] [General]
Здравствуйте
Подскажите как подключиться к Меркурий 200 по CAN линии?

**Comment by VisualGadget** [comment]
Здравствуйте.
Проект работает сейчас в железе и редкие коммиты появляются при необходимости.

Для подключения лучше использовать CAN адаптер, или как я сделал, RS-485, но тогда придётся уровни сигнала сдвигать. В документации описано [что я сделал](https://github.com/VisualGadget/Mercutel/blob/main/doc/%D0%A1%D0%B1%D0%BE%D1%80%D0%BA%D0%B0/%D0%90%D0%BF%D0%BF%D0%B0%D1%80%D0%B0%D1%82%D0%BD%D1%8B%D0%B5%20%D0%BC%D0%BE%D0%B4%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%86%D0%B8%D0%B8%20%D0%B1%D0%BB%D0%BE%D0%BA%D0%BE%D0%B2.md).
